modules/dnn/src/tensorflow/tf_importer.cpp

```diff
@@ -1462,18 +1462,22 @@ void TFImporter::populateNet(Net dstNet)
             connect(layer_id, dstNet, parsePin(layer.input(1)), id, 0);
             data_layouts[name] = DATA_LAYOUT_UNKNOWN;
         }
-        else if (type == "ResizeNearestNeighbor")
+        else if (type == "ResizeNearestNeighbor" || type == "ResizeBilinear")
         {
             Mat outSize = getTensorContent(getConstBlob(layer, value_id, 1));
             CV_Assert(outSize.type() == CV_32SC1, outSize.total() == 2);
 
             layerParams.set("height", outSize.at<int>(0, 0));
             layerParams.set("width", outSize.at<int>(0, 1));
+            if (type == "ResizeNearestNeighbor")
+                layerParams.set("interpolation", 0);
```

Comment:
I think `INTER_NEAREST` or `INTER_LINEAR` should be here.

Reply:
Hi, do you mean replacing the values by them? Then I should define them at the beginning or include imgproc.hpp, right?

Reply:
Why? They are just defined in OpenCV: https://docs.opencv.org/master/da/d54/group__imgproc__transform.html#ga5bb5a1fea74ea38e1a5445ca803ff121 . And please do not forget to add a test.